File: Assignment/Module_2/2.3/factorial_using_recursion.c
#include<stdio.h>
#include<conio.h>

int factorial(int num){
	
	if(num == 1 || num == 0){
		return 1;
	}

	return num * factorial(num - 1);
}


void main(){
	
	int num, result;
	
	printf("Enter number : ");
	scanf("%d", &num);
	
	result = factorial(num);
	
	printf("Factorial number of %d is %d", num, result);
}

File: Python/factorial_using_recursion.py
def factorial(num) :
    if num == 0 or num == 1 :
        return 1
    else :
        return num*factorial(num-1)

a = int(input("Enter Number :"))
print("Factorial of " ,a ,"is :", factorial(a))
#print(type(factorial(50)))
